internal/util/secretskms.go

```diff
@@ -0,0 +1,224 @@
+/*
+Copyright 2019 The Ceph-CSI Authors.
```

Comment:
@nixpanic 2019/2021 ?

Reply:
it is just moving the existing contents to a new file, and adding more things there, not sure it needs an update

Reply:
true, I was thinking the copyright can be new as its new file, but current version is also fine.